src/library_pthread.js

```diff
@@ -402,8 +402,18 @@ var LibraryPThread = {
 #if PTHREADS_DEBUG
         out('Allocating a new web worker from ' + new URL('{{{ PTHREAD_WORKER_FILE }}}', import.meta.url));
 #endif
-        // Use bundler-friendly `new Worker(new URL(..., import.meta.url))` pattern; works in browsers too.
+#if TRUSTED_TYPES
+        // Use Trusted Types compatible wrappers.
+        if (typeof trustedTypes !== 'undefined' && trustedTypes.createPolicy) {
+          var p = trustedTypes.createPolicy('emscripten#workerPolicy1', { createScriptURL: function(ignored) { return new URL('{{{ PTHREAD_WORKER_FILE }}}', import.meta.url)} });
```

Comment:
We don't have strict rules on line length, but it would be nice to avoid a super-long one here I think, maybe like this:

```suggestion
          var p = trustedTypes.createPolicy('emscripten#workerPolicy1', {
            createScriptURL: function(ignored) {
              return new URL('{{{ PTHREAD_WORKER_FILE }}}', import.meta.url);
            }
          });
```